src/types.ts

```diff
@@ -25,13 +26,23 @@ export type PathsOutput = string[];
 
 export type Output = OnlyCountsOutput | PathsOutput | GroupOutput;
 
+export type FSLike = {
```

Comment:
this is purposely explicit rather than just `FSLike = typeof nativeFs` since the `fs` module has all sorts of exports (both types and actual exports) which we don't want to have to stub out to satisfy the type when bringing our own fs-like object